### PR TITLE
 Fixed highligting of playing track

### PR DESCRIPTION
**Pull request by artandtechvof**
Fixed highligting of playing track

### CODE REVIEW

File: app.html [hunk 1]
```diff
@@ -636,15 +637,17 @@
...
-//  const preset_tags = document.getElementsByClassName('track');
-//  for (var i = 0; i < track_lines.length; i++) {
-//    if (track_lines[i] == track) {
-//      track_tags[i].style.background = '#505080';
-//    } else {
-//      track_tags[i].style.background = '#101040';
-//    }
-//  }
+// FIXME -> fixed!!
+ if (track_string != ""){
```
Comment (profezzorn) [inline app.html]: add space between ) and {

File: app.html [hunk 2]
```diff
@@ -636,15 +637,17 @@
...
-// FIXME
-//  const preset_tags = document.getElementsByClassName('track');
-//  for (var i = 0; i < track_lines.length; i++) {
-//    if (track_lines[i] == track) {
-//      track_tags[i].style.background = '#505080';
-//    } else {
-//      track_tags[i].style.background = '#101040';
-//    }
-//  }
+// FIXME -> fixed!!
```
Comment (profezzorn) [inline app.html]: just remove this comment

Comment (artandtechvof) [reply app.html]: done

File: app.html (hunk 1 above)
Comment (artandtechvof) [reply app.html]: done

**Pull request merged**